src/intl/messages.js

```diff
@@ -566,10 +584,15 @@ export const messages: { [messageId: string]: MessageType } = {
   selectTheBootableDeviceTooltip: 'Select the bootable devices in the order you would like them to boot.',
   sendShortcutKey: 'Send Key',
   sendCtrlAltDel: 'Ctrl+Alt+Del',
+  serialConsole: 'Serial Console',
+  serialConsoleOptions: 'Serial Console Options',
+  serverStorage: 'Server-side',
+  serverStorageTooltip: 'Settings are persisted on the server.',
   sessionExpired: {
     message: 'Your session is about to timeout due to inactivity.',
     description: 'Primary message for SessionTimeout modal component',
   },
+  settingsStorage: 'Settings storage',
```

Comment:
Why we need this?

Reply:
I plan to use it as section under Advanced Options to allow opt-out (currently only language).

Reply:
I prefer a different label, more user friendly, like "persist account settings". This will require a tooltip as well....

Reply:
How about Advanced Options -> Storage -> Persist language +  checkbox or  switch?

Reply:
I don't like the "storage" label name. It's confusing with the storage settings for a VM.